homeassistant/components/light/yeelight.py

```diff
@@ -85,7 +84,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             device = {'name': device_config[CONF_NAME], 'ipaddr': ipaddr}
             lights.append(YeelightLight(device, device_config))
 
-    add_devices(lights, True)  # true to request an update before adding.
+    add_devices(lights, True)
```

Comment:
Very implicitly, but can be fixed with a documentation update. Maybe enforcing using kwarg for booleans as a general policy? Would turn it to add_devices(lights, update_before_adding=True) to make it clear?